### DISCUSSION TITLE
Wind gusts

### DISCUSSION THREAD

**Discussion by markallisongit** [aeharding/ppg.report#143] [Ideas]
Love this tool, one metric that is available in the API that would be very useful for PPG pilots is wind gusts. Is it possible to add that as an optional feature for display?

**Comment by aeharding** [comment]
Hi Mark, wind gust is in the header, but its surface value only. Hopefully that helps!

<img width="344" alt="image" src="https://github.com/aeharding/ppg.report/assets/2166114/d16f0755-0d67-408e-bc4d-4acebb3d1af3">
